src/components/Slide/SlideBase/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+import { Slide as MaterialSlide, SlideProps as MaterialSlideProps } from "@mui/material";
+
+export interface SlideBaseProps extends MaterialSlideProps {
+    unmountOnExit?: boolean;
+    mountOnEnter?: boolean;
```

Comment:
I'm not sure if it's necessary to add this?

Reply:
Okay, I removed it.